contents/docs/llm-analytics/troubleshooting.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Troubleshooting and FAQs
+---
+
+
+## How much does LLM analytics cost?
+
+Your first 100,000 `$ai_model` events each month are free – i.e. if you never exceed this number, you can use LLM analytics for free.
+
+After this, we charge a small amount for each `$ai_*` event you send.
+
+Go to the [pricing page](/pricing) to use our calculator to get an estimate. You can also view an estimate on [your billing page](https://us.posthog.com/organization/billing).
+
+## Why can't I see any of my LLM events?
```

Comment:
It might be worth making this a snippet and just using it instead, there are a looooot of reasons why events might not show up.

https://posthog.com/docs/product-analytics/troubleshooting#why-are-events-not-appearing-in-my-project